run-tests.sh

```diff
@@ -158,18 +158,18 @@ ARTIFACTS_DIR="$WORKSPACE/artifacts"
 mkdir -p "$ARTIFACTS_DIR"
 
 create_env_file || exit 1
-docker cp "$TESTS_ENV_FILE" "$TEST_CONTAINER_ID:/opt/app-root/src/tests/env.yaml"
+docker cp "$TESTS_ENV_FILE" "$TEST_CONTAINER_ID:/opt/app-root/tests/env.yaml"
 
 # tests
 echo '===================================='
 echo '===     Running Tests           ===='
 echo '===================================='
 set +e
-docker exec "$TEST_CONTAINER_ID" /bin/bash -c "pytest --junitxml=test-report.xml"
+docker exec "$TEST_CONTAINER_ID" /bin/bash -c "pytest --junitxml=test-report.xml tests"
 TEST_RESULT=$?
 set -e
 # Copy test reports
-docker cp "$TEST_CONTAINER_ID:/opt/app-root/src/test-report.xml" "$WORKSPACE/artifacts/junit-test-report.xml"
```

Comment:
same as before, we could use relative routes to avoid future path changes.

Reply:
it should definitely work: 

An excerpt of [Docker cd documentation](https://docs.docker.com/engine/reference/commandline/cp/#extended-description):
```
The docker cp command assumes container paths are relative to the container’s / (root) directory. This means supplying the initial forward slash is optional; The command sees compassionate_darwin:/tmp/foo/myfile.txt and compassionate_darwin:tmp/foo/myfile.txt as identical. Local machine paths can be an absolute or relative value. The command interprets a local machine’s relative paths as relative to the current working directory where docker cp is run.
```

What version of Docker cli are you using ? that may be the issue there , but not sure.

Reply:
Well, it works for me locally :see_no_evil: show the documentation to Jenkins :laughing: